Dockerfile

```diff
@@ -1,14 +1,11 @@
-#
-# Build and copy GraphQL binaries and run them.
-#
-# Version values referenced from https://hub.docker.com/_/microsoft-dotnet-aspnet
-FROM mcr.microsoft.com/dotnet/sdk:6.0 as build
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0. AS build
+
 
 WORKDIR /src
-COPY [".", "./"]
+COPY . .
 RUN dotnet build "./src/Service/Azure.DataApiBuilder.Service.csproj" -c Docker -o /out -r linux-x64
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0 as runtime
+FROM mcr.microsoft.com/cbl-mariner/base/core:2.0 AS runtime
```

Comment:
Could you pleaser use this image to stay compatible to the old one? `mcr.microsoft.com/dotnet/aspnet:6.0-cbl-mariner2.0`